packages/core/src/providers/linkedin.ts

```diff
@@ -83,36 +71,12 @@ export default function LinkedIn<P extends LinkedInProfile>(
   return {
     id: "linkedin",
     name: "LinkedIn",
-    type: "oauth",
-    authorization: {
-      url: "https://www.linkedin.com/oauth/v2/authorization",
-      params: { scope: "r_liteprofile r_emailaddress" },
-    },
-    token: "https://www.linkedin.com/oauth/v2/accessToken",
+    type: "oidc",
+    wellKnown:
+      "https://www.linkedin.com/oauth/.well-known/openid-configuration",
```

Comment:
```suggestion
```

Reply:
@ThangHuuVu @HugeLetters 

This suggestion was wrong, the default wellKnown of `https://www.linkedin.com/.well-known/openid-configuration` does not work!

![Screenshot 2024-03-09 at 6 44 38 AM](https://github.com/nextauthjs/next-auth/assets/543981/5bf90523-1198-4a23-9489-423b084e1604)